beater/pubsubbeat.go

```diff
@@ -100,6 +105,46 @@ func (bt *Pubsubbeat) Run(b *beat.Beat) error {
 		bt.pubsubClient.Close()
 	}()
 
+	events := make(chan ackableEvent)
+
+	go func() {
+		// batch up documents to prevent contending on lock in client.Publish()
+
+		batchMaxSize := 512
+		batchMaxTime := 1 * time.Second
```

Comment:
These are currently hardcoded, I can turn them into config options if needed.

Reply:
Hard coded seems reasonable for a first pass.